Remotion/Data/DomainObjects.UnitTests/Data.DomainObjects.UnitTests.csproj

```diff
@@ -32,6 +32,7 @@
     <ProjectReference Include="..\..\Development\Mixins\Development.Mixins.csproj" />
     <ProjectReference Include="..\..\Development\Moq\Development.Moq.csproj" />
     <ProjectReference Include="..\..\Mixins\Core\Mixins.Core.csproj" />
+    <ProjectReference Include="..\DomainObjects.UberProfIntegration\Data.DomainObjects.UberProfIntegration.csproj" />
```

Comment:
Should we keep this dependency or clean it up before the merge. normally, this dependency would be unwanted since it breaks the project hierarchy for the unit tests. on the other hand, it doesn't hurt us and is convenient when doing profiling.

Reply:
> [...] this dependency would be unwanted since it breaks the project hierarchy for the unit tests.

How does this project reference break the project hierarchy? 

> on the other hand, it doesn't hurt us and is convenient when doing profiling.

I left it in for that reason. Because we are referencing the profiler assemblies only indirectly we do not introduce a new dependency while making it more convenient to enable the profiler.